### PR TITLE
Packaging Adjustments

### PR DESCRIPTION
**Pull request by lippserd**
* Rename `COPYING` to `LICENSE`
* Adjust `.gitattributes` for export ignore
* Schema path is now always `/usr/share/icingaweb2/schema`
* No longer offer documentation to install SELinux policy manually since all RPM-based distros provide the policy package
* Remove `packages/` as it has been moved to the packaging repository

### CODE REVIEW

File: modules/setup/library/Setup/WebWizard.php [hunk 1]
```diff
@@ -445,7 +445,7 @@ public function getSetup()
                         ? $pageData['setup_auth_db_creation']['password']
                         : null,
                     'schemaPath'        => Config::module('setup')
-                        ->get('schema', 'path', Icinga::app()->getBaseDir('etc' . DIRECTORY_SEPARATOR . 'schema'))
+                        ->get('schema', 'path', Icinga::app()->getBaseDir('schema'))
```
Comment (nilmerg) [inline modules/setup/library/Setup/WebWizard.php:448]: This location does not exist. The schema files are still located in `etc/schema`.

**Review by nilmerg — CHANGES_REQUESTED** [review]

File: modules/setup/library/Setup/WebWizard.php (hunk 1 above)
Comment (lippserd) [reply modules/setup/library/Setup/WebWizard.php:448]: Done.

File: doc/80-Upgrading.md [hunk 2]
```diff
@@ -13,8 +13,9 @@ v2.6 to v2.8 requires to follow the instructions for v2.7 too.
 **Database Schema**
 
 * Please apply the `v2.11.0.sql` upgrade script depending on your database vendor.
-  In package installations this file can be found in `/usr/share/doc/icingaweb2/schema/*-upgrades/`
-  (Debian/Ubuntu: `/usr/share/icingaweb2/etc/schema/*-upgrades/`).
+  As of version `2.11.3`, upgrade scripts can be found at `/usr/share/icingaweb2/schema/*-upgrades/`.
+  Older versions install these files to `/usr/share/doc/icingaweb2/schema/*-upgrades/` for RPM-based systems 
+  and `/usr/share/icingaweb2/etc/schema/*-upgrades/` for Debian or Ubuntu.
```
Comment (nilmerg) [inline doc/80-Upgrading.md:18]: This still references etc, is this correct?

Comment (lippserd) [reply doc/80-Upgrading.md:18]: Yes since it is for "older versions".

Comment (nilmerg) [reply doc/80-Upgrading.md:18]: Oh. :elephant:

**Review by nilmerg — APPROVED** [review]

**Comment by jeredfloyd** [comment]
Hello! Where is the "packaging repository" to which the SELinux policy files have moved? I can't find it on GitHub, and there doesn't seem to be a 2.11.4 tag or recent updates at https://git.icinga.com/packaging/rpm-icingaweb2

**Comment by sebastic** [comment]
> Remove packages/ as it has been moved to the packaging repository

Why don't the git.icinga.com repos reflect the source packages?

**Comment by lbetz** [comment]
Was moving the schemes really necessary? Anyone who works with automation is treated badly. No advance notice, no compatibility - nothing. All done in a 2.11.x release.

**Comment by miken32** [comment]
@lbetz not to mention anyone that uses seliux. sure this software is open source, but guess you'll have to pay if you want to make it useful.

**Pull request merged**